base/src/dispatcher/dispatcher.ts

```diff
@@ -1053,13 +1054,14 @@ export enum RecognitionOp {
   OR,
   NOT,
```

Comment:
but you already had NOT, how is this different from PROPERTY NE?

Reply:
NOT isnt actually implemented currently, but I see not as:
A & !B -> 'A' AND 'NOT B'

where as NE is
A != B -> 'A' NOT EQUAL 'B'

NOT is unary but NE is binary